src/scancode/cli.py

```diff
@@ -461,8 +468,14 @@ def scancode(ctx,
         if not quiet:
```

Comment:
IMHO this message should come after the post scan plugin runs.
And we should also have a message when not quiet telling that we are running post scans plugins and which one